### ISSUE TITLE
Version needs to be incremented again, otherwise this package won't be picked up

### ISSUE THREAD

**Issue by fingolfin** [osj1961/guava#16]
Hi there,

to quote a report by @alex-konovalov :

> Package: GUAVA
> changed components: [ "Dependencies", "PackageWWWHome", "ArchiveURL",  "README_URL", "PackageInfoURL" ]
> ERROR (GUAVA): There are changed components in the new info file from
> http://www.southernct.edu/~fields/Guava/PackageInfo.g
> but .Version remains the same. This is not allowed, so the info file will not be changed
> PackageInfo.g is not accepted because of an error.

It seems you modified the PackageInfo.g at <http://www.southernct.edu/~fields/Guava/PackageInfo.g> without changing the version number in it. That is not allowed by the system (after all, you now assigned the same version to two clearly different things).

**Comment by osj1961** [comment]
It seems that the old version (3.12) was restored somehow -- I had certainly put the 3.13 PackageInfo.g file in that location (otherwise the fact that other changed components are different wouldn't have been recognized).  I wonder if IT restored the files on giam.southernct.edu from a backup?  I have put the latest version back.

**Comment by alex-konovalov** [comment]
Finally 3.13 picked up:
```
* GUAVA - new version 3.13 discovered!!!
  ============================================
  Getting new archives from 
  http://osj1961.github.io/guava/archives/guava-3.13[ ".tar.gz" ]
  unpacking guava-3.13.tar.gz
  Validation of the info file successful!
  Package GUAVA 3.13 from 31/01/2016 has dependencies:
  * GAP >= 4.8.0
  * SuggestedOtherPackages [ [ "SONATA", ">= 2.3" ] ]
...
```
You can close this issue now.

**Comment by osj1961** [comment]
Just found one bug fix that got left out, so 3.14 will hopefully come out shortly.  (NOT due to  triskaidekaphobia!)
